tests/server_test.go

```diff
@@ -154,7 +146,7 @@ func runServer(tb testing.TB, options *serverOptions, tests func(session *testSe
 			TestServerVersionInfo.Vendor,
 			TestServerVersionInfo.SupportURL,
 		),
-		gluon.WithStoreBuilder(&testBadgerStoreBuilder{}),
+		gluon.WithStoreBuilder(&store.BadgerStoreBuilder{}),
```

Comment:
The Test badger store function also suppresses the badger log system so that when one traces the test there no noise from badger. Could you please restore that?